src/redprl/refiner.fun

```diff
@@ -17,7 +17,7 @@ struct
 
   structure Hyp =
   struct
-    fun Project z alpha jdg =
+    fun Project z _ jdg =
```

Comment:
If you feel very strongly about it, then `_` is at least preferable to `_(*alpha*)`.

I will just say that I understand what you are saying, but don't agree in the general case. For instance, in some places I define a dummy structure that ascribes a signature, to check if some other structure conforms to that signature (as a sanity check):

```sml
structure _MyStruct : SIG = MyStruct
```